### ISSUE TITLE
nvcc: command not found (Vitis-AI-3.0)

### ISSUE THREAD

**Issue by A7eNg** [Xilinx/Vitis-AI#1378]
After building and entering docker, met this error 
`nvcc: command not found`

![image](https://github.com/Xilinx/Vitis-AI/assets/48305976/848feebe-79eb-4284-8c60-434d49130a5c)
 And there is no bin file folder in path: /usr/local/cuda-11.3/
![image](https://github.com/Xilinx/Vitis-AI/assets/48305976/93acb924-848f-4132-a65d-ff006d9d4b6c)

How to fix it in Vitis-AI-3.0? I have not met this problem in version 3.5, but i am using MPSoC, so whether 3.5 can be used for MPSoC devices？Or I must change to 3.0?

Thanks for your kind reply!

**Comment by lishixlnx** [comment]
Yes, you must change to v3.0.  
MPSoC is only supported to v3.0.
If you want to use cuda, can you try to install the missing part manually?

**Comment by A7eNg** [comment]
Thanks for your comment, should I use "opt_pytorch" for the target framework?

**Comment by cchalou98** [comment]
Hello, please did you solve this issue? cuz am having the same issue? I am using Vitis 3.0 to quantize yolov5, Ihave GPU NVIDIA GeForce RTX 4060, and am using CUDA 11.3.1 in vitis?
![Screenshot from 2023-12-19 10-45-51](https://github.com/Xilinx/Vitis-AI/assets/99268986/f0c1d507-e153-4724-9849-f9ef4a0e7eea)

**Comment by A7eNg** [comment]
Yeah, I have solved this issue by installing CUDA 11.3 and cudnn manually

**Comment by cchalou98** [comment]
> Yeah, I have solved this issue by installing CUDA 11.3 and cudnn manually

Could you please share with me the command lines how you installed it?? much appreciated, thank you.
